v2/tools/generator/internal/armconversion/convert_from_arm_function_builder.go

```diff
@@ -47,9 +48,10 @@ func newConvertFromARMFunctionBuilder(
 			armTypeIdent:          c.armTypeName.Name(),
 			idFactory:             c.idFactory,
 			typeKind:              c.typeKind,
+			payloadType:           c.payloadType,
 			codeGenerationContext: codeGenerationContext,
 		},
-		typeConversionBuilder: astmodel.NewConversionFunctionBuilder(c.idFactory, codeGenerationContext),
+		typeConversionBuilder: astmodel.NewConversionFunctionBuilder(c.idFactory, codeGenerationContext, false),
```

Comment:
Is there a better option than passing a mysterious **false** here?

Reply:
I've changed this somewhat so I think it's clearer